spec/models/post_spec.rb

```diff
@@ -25,6 +25,12 @@
     expect(post).to_not be_valid
   end
 
+  it 'is not valid with a negative likes_counter' do
+    user = User.create(name: 'John', posts_counter: 0)
+    post = user.posts.build(title: 'Negative Likes', comments_counter: -1, likes_counter: 0)
+    expect(post).to_not be_valid
+  end
+
```

Comment:
- [ ] Great work attempting to test the validity of the `likes_counter` attribute of the `Post` model. However, in the newly written test, you have only tested the `comments_counter` and not the `likes_counter`. Therefore, you have written two of a kind for the `comments_counter`. 👍 💻 

#### N.B: Please ensure that all tests pass after making the necessary corrections. 🖥️ 🖱️